### DISCUSSION TITLE
Why does next images lazy loading not use the loading tag inside image ? does this not impact SEO ?

### DISCUSSION THREAD

**Discussion by apiki-wp-care** [vercel/next.js#36782] [Help]
Hey guys. I am doing a next.js project and I see that the img tag generated does the lazyloading without using the loading='lazy' inside the image. How next do the lazyloading without it and does this approach without the loading='lazy' hurt SEO or google bot's content recognition in any way?

**Comment by icyJoseph** [comment]
If JavaScript is not enabled, there's a `noscript` tag as last child to the `next/image` rendered DOM that handles lazy loading. 

Otherwise, if it can run JavaScript, the image is hot swapped and loaded, once the viewport intersects with it. 

So my assumption here is that if a Bot makes a request for your page, and the bot doesn't do a headless JS run scrolling down the page, then they'd still see the `noscript` tag, and use the image from there. 

Why exactly this is done with intersection observers, I don't remember, I tried to find a video from back when `next/image` was introduced, but no luck.
